packages/action-password-update/src/components/PasswordUpdateDialog/PasswordInput.js

```diff
@@ -10,6 +10,7 @@ const PasswordInput = props => (
       value={props.value}
       onChange={e => props.onChange(e.target.value)}
       readOnly={props.readOnly === true}
+      disabled={props.readOnly === true}
```

Comment:
My intention was do prevent focus readonly fields by using tab key. If an user want to add new informations this is advantage, if an user want to copy data from a readonly field this is a disadvantage. My personal choice is the first option.

From my perspective sending readonly data has no point, it only takes more bandwidth.